### PR TITLE
Several new websites

### PR DESCRIPTION
**Pull request by DRSDavidSoft**
### List of domains
- `afkarnews.com`
- `digikala.com`
- `doostihaa.com`
- `emalls.ir`
- `irna.ir`
- `itbazar.com`
- `mashreghnews.ir`
- `mostafa3d.com`
- `rokna.ir`
- `rokna.net`
- `salamatnews.com`
- `softgozar.com`
- `timecity.ir`

### CODE REVIEW

**Comment by DRSDavidSoft** [comment]
@farrokhi Sorry this PR has been open for too long. Can you take a look, please?

File: filter.txt [hunk 1]
```diff
@@ -949,6 +964,7 @@ zoomtech.ir###text-3
 ||hamgardi.com/HamgardiImages/HamgardiAds/$image
 ||harmonydl.info/wp-content/themes/harmony1.11/js/popup1.js
 ||harmonydl.info/wp-content/uploads/*/*/banner
+||hexdownload.info/img/ppop.js
```
Comment (farrokhi) [inline filter.txt:967]: this line is duplicate.

File: filter.txt [hunk 2]
```diff
@@ -699,6 +713,7 @@ tigatravel.com##.fade
 tigatravel.com##.popup_block
 timecity.ir##.banner
 tinn.ir##a[href^="/advertisements/"]
+timecity.ir##.banner
```
Comment (farrokhi) [inline filter.txt:716]: Duplicate. See two lines above.

File: filter.txt [hunk 3]
```diff
@@ -1192,6 +1209,7 @@ zoomtech.ir###text-3
 ||sigmaad.com^
 ||simcart.com/a/$image
 ||skinak.ir/up/skinak/temps/forum/v3/images/bg.jpg
+||snn.ir/files/adv/$image
```
Comment (farrokhi) [inline filter.txt:1212]: Already covered by `/files/adv/*` rule

File: filter.txt [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [Adblock Plus 2.0]
-! Checksum: i2FafaULYy879EXb1fbLyw
+! Checksum: dsBkvHv0ssUAdbpg26roZA
 !--------------------------------------------------------------------------
 ! Title :    Adblock-Iran
 ! Expires:   4 days
```
Comment (farrokhi) [inline filter.txt:5]: please bump version

**Review by farrokhi — CHANGES_REQUESTED** [review]

**Comment by DRSDavidSoft** [comment]
@farrokhi Sorry about duplicate lines, bumped the version and updated the checksum.
Any other changes?

**Review by farrokhi — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**